src/zcash/IncrementalMerkleTree.hpp

```diff
@@ -246,19 +233,31 @@ class PedersenHash : public uint256 {
 template<size_t Depth, typename Hash>
 EmptyMerkleRoots<Depth, Hash> IncrementalMerkleTree<Depth, Hash>::emptyroots;
 
-} // end namespace `libzcash`
+} // namespace libzcash
 
-typedef libzcash::IncrementalMerkleTree<INCREMENTAL_MERKLE_TREE_DEPTH, libzcash::SHA256Compress> SproutMerkleTree;
-typedef libzcash::IncrementalMerkleTree<INCREMENTAL_MERKLE_TREE_DEPTH_TESTING, libzcash::SHA256Compress> SproutTestingMerkleTree;
+typedef libzcash::IncrementalMerkleTree<INCREMENTAL_MERKLE_TREE_DEPTH, libzcash::SHA256Compress>
+    SproutMerkleTree;
+typedef libzcash::
```

Comment:
I'd prefer a penalty for breaking here, but I don't think there's an option for that.